lib/api/initiateMultipartUpload.js

```diff
@@ -258,47 +260,50 @@ function initiateMultipartUpload(authInfo, request, log, callback) {
 
     metadataValidateBucketAndObj(metadataValParams, log,
```

Comment:
Just for the sake of readability I believe it could be time to use a helper like `async.waterfall` to limit indentation levels